app/models/library_entry.rb

```diff
@@ -99,9 +99,15 @@ def activity
   end
 
   before_save do
-    if status_changed? && status == :completed && media.progress_cap
+    if status_changed? && status == 'completed' && media.progress_limit
       # When marked completed, we try to update progress to the cap
-      self.progress = media.progress_cap
+      self.progress = media.progress_limit
+    elsif progress == media.progress_limit
+      # when in current and episodes changed to 25
```

Comment:
This comment is cruel, evil stuff, please don't do that.

It's not "when it changes to 25"